options/locale/locale_zh-CN.ini

```diff
@@ -1747,6 +1747,7 @@ settings.add_telegram_hook_desc=将 <a href="%s">Telegram</a> 集成到您的仓
 settings.add_matrix_hook_desc=将 <a href="%s">Matrix</a> 集成到您的仓库中。
 settings.add_msteams_hook_desc=将 <a href="%s">Microsoft Teams</a> 集成到您的仓库中。
 settings.add_feishu_hook_desc=将 <a href="%s">Feishu</a> 集成到您的仓库中。
+settings.add_wechatwork_hook_desc=将 <a href="%s">Wechatwork</a> 集成到您的仓库中。
```

Comment:
these translation files are maintained on crowdin, Just add this new line in ``local_en-US.ini`` is enough, Thanks.

Reply:
I have made modifications